docs/conf.py

```diff
@@ -1,13 +1,16 @@
 # -*- coding: utf-8 -*-
-import sys, os
+import sys, os, django
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.append(os.path.abspath('.'))
 sys.path.append(os.path.abspath('..'))
 sys.path.append(os.path.abspath('../tests'))
+sys.path.insert(0, os.path.abspath(os.path.join('..', '..', 'cookie_consent')))
```

Comment:
I'd use `pathlib.Path` here instead, which is the modern way to deal with filesystem paths:

```py
root_dir = pathlib.Path(__file__).parent.parent
sys.path.insert(0, root_dir.resolve())
```
adding `cookie_consent` is not correct, as it's the parent directory that needs to be added in the sys path (which line 8 is supposed to do)

Reply:
What you propose seems to work:

```
import pathlib
root_dir = pathlib.Path(__file__).parent.parent
sys.path.insert(0, root_dir.resolve())
```

This seems to work too:

`sys.path.append(os.path.abspath('../cookie_consent'))`

I am fine with whichever one the maintainers decide to use.